packages/expo-contacts/src/Contacts.ts

```diff
@@ -1,13 +1,10 @@
-// @flow
-
-// TODO: Bacon: No React Native
-import { Share } from 'react-native';
-import { Platform } from 'expo-core';
+import { UnavailabilityError } from 'expo-errors';
+import { Platform, Share } from 'react-native';
 import UUID from 'uuid-js';
+
 import ExpoContacts from './ExpoContacts';
-import { UnavailabilityError } from 'expo-errors';
 
-type CalendarFormatType =
+export type CalendarFormatType =
```

Comment:
The changes to these type definitions look OK for keeping the existing API the same, but I think we should rehaul all of them in a future PR to just be enums instead of `type XType` + `const Xs = { ... }`.